en/middleware/micrortps_manual_code_generation.md

```diff
@@ -133,3 +133,7 @@ src/modules/micrortps_bridge/micrortps_client
 └── microRTPS_transport.h
  0 directories, 4 files
 ```
+
+## Build and use the code
```

Comment:
@VicenteEprosima Can we expand on this a bit. In #210 you said that once automatic code generation is disabled you can just use "normal build processes" for the client and agent, BUT in the normal build process we don't build the client (it is done for us). I don't think just running the normal build process will do the compile set for us because the  manually generated source code is not located at the same location as the automatically generated source code. 

1. How do we build the client manually? (command line we enter please - and does it differ for SITL vs nuttx vs snapdragon?)

It does seem likely that the firmware build process will automatically generate the firmware and include the client executables IFF they are present in the normal place - which they should be after manually building.

2. Can you confirm?

3. Do we have to do anything different for building the agent other than use the source from a different location?